README.md

```diff
@@ -32,33 +32,78 @@ ShapeShift's OSS 2nd generation Web application. (Under Development)
 - [shapeshift](https://shapeshift.com/developer-portal)
 
 ## Dependencies
-
 - [hdwallet](https://github.com/shapeshift/hdwallet)
 - [lib](https://github.com/shapeshift/lib)
 - [unchained](https://github.com/shapeshift/unchained)
+- [nvm](https://github.com/nvm-sh/nvm#installing-and-updating) (optional; must be installed manually)
 
-## Developer Onboarding
+## Quick Start
 
 On Linux and MacOS it works out of the box following the steps.<br/>
 ⚠️ On Windows you should use the _Windows Subsystem for Linux_ (WSL).
 
-- Copy sample env file:
+- Clone the repo
+
+- (optional) Make sure you're using the right Node.js version.
 
   ```sh
-  cp sample.env .env
+  nvm use
   ```
+The .env file contains global variables that the program needs to function properly. The following variables are not publicly available for security reasons:
+
+REACT_APP_PORTIS_DAPP_ID<br>
+Allows you to connect a Portis wallet. Without this the program will hang after choosing Portis and 
+clicking the "Pair" button. To get it, make a post asking for it in the Discord.
+
+REACT_APP_ETHEREUM_NODE_URL <br>
+Program crashes when connecting wallet without this. Obtain by doing the following: 
+1. Go to https://infura.io/dashboard
+2. Set up a free account
+3. Make a new project 
+<br>Your key should use "JSON-RPC over HTTPS" and look like this: 
+https://mainnet.infura.io/v3/YOUR-PROJECT-ID
 
 - Install Dependencies:
 
   ```sh
+  # This is short for `yarn install`; be sure to use `yarn install --frozen-lockfile` instead if you're setting up a CI pipeline or trying to duplicate a historical build.
   yarn
   ```
 
-- Run server
+- Copy `sample.env` to `.env`, and configure it according to the [.env section](#.env) below.
+
+  ```sh
+  cp sample.env .env
+  ```
+
+- Have fun!
   ```sh
+  # Launch a local dev-mode server with hot-reload support:
   yarn dev
+  # Or, compile a static production version:
+  yarn build
   ```
 
+### `.env`
+
+The `.env` file contains environment variables that the program needs to function properly. Some of these variables are deployment-specific, so they aren't included in the repository.
+
+- `REACT_APP_PORTIS_DAPP_ID`
+
+  Allows you to connect a Portis wallet. Without this the program will hang after choosing Portis and clicking the "Pair" button. Portis Dapp IDs aren't secret, but they are domain-specific; you can get the one we use for testing by making a post asking for it in the [Discord](https://discord.gg/shapeshift).
+
+- `REACT_APP_ETHEREUM_NODE_URL`
+
+  Needed for certain Defi integrations such as Yearn; the app will malfunction when connecting a wallet without it.
+
+  Any Ethereum node should do, but you can get your own node URL for testing by doing the following:
+
+  1. Go to https://infura.io/dashboard
+  2. Set up a free account
+  3. Make a new project
+
+      Your key should use "JSON-RPC over HTTPS" and look like `https://mainnet.infura.io/v3/<your project id>`
```

Comment:
this is duplicated from above, we should only include in one place

Reply:
Dammit, flipping automerge decided to leave the duplicate stuff.

Reply:
This is the copy that should stay, https://github.com/shapeshift/web/pull/562#discussion_r774692867 should nuke the nukable one